lib/pf/locationlog.pm

```diff
@@ -185,7 +186,7 @@ sub locationlog_db_prepare {
             LEFT JOIN node USING (mac)
         WHERE switch=? AND port=?
             AND (node.voip='' OR node.voip='no')
-            AND (ISNULL(end_time) OR end_time = 0)
+            AND end_time = 0
```

Comment:
To space or not to space, yes that is the question.

I personally prefer no space but as long as it is consistent in the file, it is ok with me